packages/augur-ui/src/modules/market-cards/market-card.tsx

```diff
@@ -177,140 +201,125 @@ export default class MarketCard extends React.Component<
-                reportingWindowEndTime={disputeInfo.disputeWindow.endTime}
-                alignRight
+                {!condensed && (
+                  <LabelValue
+                    label="Open Interest"
+                    value={openInterestFormatted.formatted}
+                    condensed
+                  />
+                )}
+              </>
+            )}
+            {reportingState !== REPORTING_STATE.PRE_REPORTING && (
+              <LabelValue
+                condensed
+                label="Total Dispute Stake"
+                value={formatAttoRep(disputeInfo.stakeCompletedTotal).formatted}
               />
-            </div>
+            )}
+            <MarketProgress
+              reportingState={reportingState}
+              currentTime={currentAugurTimestamp}
+              endTimeFormatted={endTimeFormatted}
+              reportingWindowEndTime={disputeInfo.disputeWindow.endTime}
+              alignRight
+            />
+          </div>
 
-            <MarketLink id={id}>
-              {description}
-            </MarketLink>
-            {!condensed && !marketResolved ?
-              <>
-                <OutcomeGroup
-                  outcomes={outcomesFormatted}
-                  marketType={marketType}
-                  scalarDenomination={scalarDenomination}
-                  min={minPriceBigNumber}
-                  max={maxPriceBigNumber}
-                  expanded={expandedView ? true : s.expanded}
-                  reportingState={reportingState}
-                  stakes={disputeInfo.stakes}
-                  dispute={dispute}
-                  inDispute={inDispute}
-                  showOutcomeNumber={showOutcomeNumber}
-                  canDispute={canDispute}
-                  canSupport={canSupport}
-                />
-                {outcomesFormatted && outcomesFormatted.length > showOutcomeNumber && !expandedView &&
+          <MarketLink id={id}>{description}</MarketLink>
+          {!condensed && !marketResolved ? (
+            <>
+              <OutcomeGroup
+                outcomes={outcomesFormatted}
+                marketType={marketType}
+                scalarDenomination={scalarDenomination}
+                min={minPriceBigNumber}
+                max={maxPriceBigNumber}
+                expanded={expandedView ? true : s.expanded}
+                reportingState={reportingState}
+                stakes={disputeInfo.stakes}
+                dispute={dispute}
+                inDispute={inDispute}
+                showOutcomeNumber={showOutcomeNumber}
+                canDispute={canDispute}
+                canSupport={canSupport}
+              />
+              {outcomesFormatted &&
+                outcomesFormatted.length > showOutcomeNumber &&
+                !expandedView && (
                   <button onClick={this.expand}>
                     <ChevronFlip
-                      stroke='#fff'
+                      stroke="#fff"
```

Comment:
formatter is off, we are using single quotes

Reply:
ah, used prettier, will adjust

Reply:
we all are using prettier. the config file has use single quotes

Reply:
in prettier.config
```
module.exports = {
  singleQuote: true,
  trailingComma: 'es5',
};

```